SurveyAppTests/Tests/Modules/Login/LoginInteractorSpec.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  LoginInteractorSpec.swift
+//  SurveyApp Tests
+//
+//  Created by Chananchida F. on 6/17/21.
+//
+
+import Quick
+import Nimble
+
+@testable import SurveyApp
+
+final class LoginInteractorSpec: QuickSpec {
+
+    override func spec() {
+        var interactor: LoginInteractor!
+        var output: LoginInteractorOutputMock!
+        var authenticationService: AuthenticationServiceProtocolMock!
+        var userSessionProvider: UserSessionProviderProtocol!
+
+        describe("a LoginInteractor") {
+            beforeEach {
+                output = LoginInteractorOutputMock()
+                authenticationService = AuthenticationServiceProtocolMock()
+                userSessionProvider = UserSessionProviderProtocolMock()
+
+                interactor = LoginInteractor(
+                    authenticationService: authenticationService,
+                    userSessionProvider: userSessionProvider
+                )
+                interactor.output = output
+            }
+
+            describe("its authenticateEmail is called") {
+                context("when the request returns success") {
+                    beforeEach {
+                        authenticationService.authenticateEmailEmailPasswordCompletionClosure = { _, _, closure in
```

Comment:
In our project we use 
```suggestion
                        authenticationService.authenticateEmailEmailPasswordCompletionClosure = { _, _, completion in
```
What do you think. Since it's a very small difference, up to you.

Reply:
I'll make that change 🙇‍♀️